### PR TITLE
docs: release 0.17.0

### PR DESCRIPTION
**Pull request by seve**
#### Reviewers
**Functional:** 

**Readability:** 

---

## Changes
- add
- remove
- modify

### CODE REVIEW

**Comment by seve** [comment]
@MDunitz Do we also want to be keeping track of the version # in the hosted_czi server?

File: .bumpversion.cfg [hunk 1]
```diff
@@ -1,9 +1,6 @@
 [bumpversion]
-current_version = 0.16.7
+current_version = 0.17.0
 commit = True
-# The below regex details an acceptable version number by naming the groups (major, minor, patch, prerel, and
```
Comment (MDunitz) [inline .bumpversion.cfg:4]: why was this removed?

**Comment by MDunitz** [comment]
> @MDunitz Do we also want to be keeping track of the version # in the hosted_czi server?

I dont think we need to track the version # in hosted_czi. For hosted the version should be based on the sha of the deployed commit

File: .bumpversion.cfg (hunk 1 above)
Comment (seve) [reply .bumpversion.cfg:4]: the cli removed it, none of these changes are manual

**Review by MDunitz — APPROVED** [review]
Lgtm
